models/sheets.js

```diff
@@ -2,68 +2,97 @@ var level = require('level');
 var uuid = require('uuid').v1;
 var extend = require('extend');
 
-module.exports = Sheets;
+var Sheets = (function() {
```

Comment:
Basically, I did two things to this file:

 - I refactored `Sheets` into a function expression so that it will be compiled at runtime. This allows us to reuse some code.
 -  I refactored the `list` function expression below so that it can be re-used for our `list` and `listAccessible` methods. These methods will show all sheets (for admins) or show only the user's accessible sheets, respectively.